apps/prairielearn/src/lib/chunks.ts

```diff
@@ -1081,7 +1039,7 @@ export const getTemplateQuestionIds = util.callbackify(getTemplateQuestionIdsAsy
  */
 export function logChunkChangesToJob(
   { updatedChunks, deletedChunks }: ChunksDiff,
-  job: Pick<Job, 'verbose'>
+  job: Pick<ServerJob, 'verbose'>
```

Comment:
I don't strongly object to this, but I feel that it's adding complexity for an only minimal improvement in safety (assuming the purpose is to stop us calling functions other than `verbose()`).

Reply:
The reason this looks like it does is that this function is being called with both legacy and modern jobs. While the types of both types of jobs are inherently incompatible, both job types *do* have a `verbose` function with an identical signature, and that's the only function that is used here. This type allows us to continue calling this function with type-safety until we completely eliminate all legacy job callers.

Reply:
Ah! I completely misunderstood the purpose! Thanks for the explanation.